http2/src/main/java/com/blazemeter/jmeter/http2/sampler/HTTP2Request.java

```diff
@@ -1,133 +1,90 @@
-    
-    protected static final String NON_HTTP_RESPONSE_CODE = "Non HTTP response code";
-
-    protected static final String NON_HTTP_RESPONSE_MESSAGE = "Non HTTP response message";
-    
+    private static final String NON_HTTP_RESPONSE_CODE = "Non HTTP response code";
+    private static final String NON_HTTP_RESPONSE_MESSAGE = "Non HTTP response message";
     public static final String IP_SOURCE = "HTTP2Request.ipSource"; // $NON-NLS-1$
-
     public static final String IP_SOURCE_TYPE = "HTTP2Request.ipSourceType"; // $NON-NLS-1$
-    
-    // Embedded URLs must match this RE (if provided)
-    public static final String EMBEDDED_URL_RE = "HTTPSampler.embedded_url_re"; // $NON-NLS-1$
-    
+    // Embedded URLs must match this regex (if provided)
+    public static final String EMBEDDED_URL_REGEX = "HTTPSampler.embedded_url_re"; // $NON-NLS-1$
     // Store MD5 hash instead of storing response
     public static final String MD5 = "HTTPSampler.md5"; // $NON-NLS-1$
-    
     public static final String EMBEDDED_RESOURCES = "HTTPSampler.embedded_resources"; // $NON-NLS-1$
-    
     public static final int SOURCE_TYPE_DEFAULT = HTTPSamplerBase.SourceType.HOSTNAME.ordinal();
-    
     public static final String ARGUMENTS = "HTTP2Request.Arguments"; // $NON-NLS-1$
-    
     public static final String POST_BODY_RAW = "HTTP2Request.postBodyRaw"; // TODO - belongs elsewhere
-
     public static final boolean POST_BODY_RAW_DEFAULT = false;
-    
     public static final String DOMAIN = "HTTP2Request.domain"; // $NON-NLS-1$
-    
     public static final String RESPONSE_TIMEOUT = "HTTP2Request.response_timeout"; // $NON-NLS-1$
-    
     public static final String FOLLOW_REDIRECTS = "HTTP2Request.follow_redirects"; // $NON-NLS-1$
-
     public static final String AUTO_REDIRECTS = "HTTP2Request.auto_redirects"; // $NON-NLS-1$
-    
-    public static final String SYNCREQUEST = "HTTP2Request.sync_request"; // $NON-NLS-1$
-
+    public static final String SYNC_REQUEST = "HTTP2Request.sync_request"; // $NON-NLS-1$
     public static final String PROTOCOL = "HTTP2Request.protocol"; // $NON-NLS-1$
-    
     public static final String REQUEST_ID = "HTTP2Request.request_id"; // $NON-NLS-1$
-    
-    /** This is the encoding used for the content, i.e. the charset name, not the header "Content-Encoding" */
+    /**
+     * This is the encoding used for the content, i.e. the charset name, not the header "Content-Encoding"
+     */
     public static final String CONTENT_ENCODING = "HTTP2Request.contentEncoding"; // $NON-NLS-1$
-
     public static final String PATH = "HTTP2Request.path"; // $NON-NLS-1$
-    
-    protected static final int MAX_FRAME_DEPTH = JMeterUtils.getPropDefault("httpsampler.max_frame_depth", 5); // $NON-NLS-1$
-    
- 
-    
-    private static Map<String, HTTP2Connection> connectionList;
-    
+
+    private static ThreadLocal<Map<String, HTTP2Connection>> connections = ThreadLocal.withInitial(HashMap::new);
```

Comment:
You can replace this initialization with `ThreadLocal.withInitial(() -> new HashMap<>());` for fix Cobertura warnings